### DISCUSSION TITLE
使用 WSL debian 时发生“参考的对象类型不支持尝试的操作”错误

### DISCUSSION THREAD

**Discussion by Exiurs** [physics-data/faq#28] [Q&A] [answered]
搜索后使用NoLsp解决该问题，然后提示 WSCSetApplicationCategory() for C:\windows\system32\wsl.exe failed with error 5

**Comment by jiegec** [comment]
你使用的windows版本是？

**Reply by Exiurs** [reply]
win11

**Comment by jiegec** [comment, marked as answer]
可以参考 https://github.com/microsoft/WSL/issues/4177 解决。
